packages/react/src/themes/teams/components/Chat/chatMessageVariables.ts

```diff
@@ -1,6 +1,9 @@
 import { pxToRem } from '../../../../lib'
 
 export interface ChatMessageVariables {
+  actionsBoxShadow: string
```

Comment:
lets agree on consistent naming for the slot variables -like `actionMenuXX`

Reply:
Done 👍